src/SymbolServer.jl

```diff
@@ -202,4 +202,14 @@ end
 
 const stdlibs = load_core()
 
+function _precompile_()
+    ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
+    Base.precompile(Tuple{Type{SymbolServer.DataTypeStore},SymbolServer.FakeTypeName,SymbolServer.FakeTypeName,Array{Any,1},Array{Any,1},Array{Symbol,1},Array{Any,1},String,Bool})
+    Base.precompile(Tuple{typeof(SymbolServer.cache_methods),Any,Dict{Symbol,SymbolServer.ModuleStore}})
+    Base.precompile(Tuple{typeof(SymbolServer.getenvtree)})
+    Base.precompile(Tuple{typeof(SymbolServer.symbols),Dict{Symbol,SymbolServer.ModuleStore}})
+    Base.precompile(Tuple{typeof(copy),Base.Broadcast.Broadcasted{Base.Broadcast.Style{Tuple},Nothing,typeof(SymbolServer._parameter),Tuple{NTuple{4,Symbol}}}})
+end
+VERSION >= v"1.4.2" && _precompile_()
```

Comment:
Is there some way to do this for old Julia versions as well?

Reply:
There is, but you risk a crash: https://github.com/JuliaLang/julia/pull/35378. I don't know whether this happens in this case because I didn't bother testing, speculating that it would be better safe than sorry.